src/main.js

```diff
@@ -28,4 +28,8 @@ exports.toJS = helpers.toJS
 exports.toImmutable = helpers.toImmutable
 exports.isImmutable = helpers.isImmutable
 
-exports.createReactMixin = require('./create-react-mixin')
+// @todo: improve how to load these addons
+// because react is defined as externals in webpack, not huge loss of space
+// if they aren't used
+exports.ReactMixin = require('../addons/react-mixin')
```

Comment:
I would rework that as:
  * move back the addons raw files into src/addons
  * create a src/addons.js that simply
```
exports.provideReact = require('./addons/provide-context')
exports.ReactMixin = require('./addons/react-mixin')
```
  * add a build target for this addons.js file that creates a index.js into ~/addons so that we can `require('nuclear-js/addons')`
  * dive into webpack configuration so that the utils file is not included 2 times (once into the nuclear-js and another into the addons), but simply one time into the nuclear-js built file